extensions/BMO/lib/Data.pm

```diff
@@ -42,6 +42,12 @@ my $mozreview_url_re = qr{
     $
 }ix;
 
+sub phabricator_uri {
+    return Bugzilla->params->{phabricator_base_uri} || 'https://example.com';
+}
+
+my $phabricator_uri = phabricator_uri();
```

Comment:
this line is still problematic, it has a lifetime of the entire process. 
Once started, it won't matter what this value is changed to using the UI, it will be whatever it was at startup.